sdk/tables/azure-data-tables/tests/test_table_batch.py

```diff
@@ -397,7 +397,7 @@ def test_batch_update_if_doesnt_match(self, resource_group, location, storage_ac
                 match_condition=MatchConditions.IfNotModified
             )
 
-            with self.assertRaises(HttpResponseError):
+            with pytest.raises(BatchErrorException):
```

Comment:
I would also make a note in the changelog about this breaking change

Reply:
Batching has not been released yet, it will be included in this upcoming preview